common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java

```diff
@@ -136,18 +135,21 @@ public void onReceivedSslError(final WebView view,
         super.onReceivedSslError(view, handler, error);
         handler.cancel();
 
+        final String sslHelpUrl = "https://docs.microsoft.com/en-us/troubleshoot/azure/active-directory/adal-authenticate-android-devices-fail";
```

Comment:
@hamiltonha Do we have guidance on including pointers to docs URLs in error messages? Do we need to create a `go.microsoft` or `aka.ms` URL for it?

Reply:
@iambmelt we need to create FWLinks, for future reference: https://review.docs.microsoft.com/en-us/help/contribute/contribute-link-to-articles-from-the-user-interface?branch=master

Reply:
Thank you!